test/cases/expressions/xhp-string-not-comment.hack

```diff
@@ -0,0 +1,11 @@
+<body> # </body>;
+
+<body> 
+# 
+</body>;
+
+<body> // </body>;
+
+<body> #hello </body>;
+
+<body>#hello</body>;
```

Comment:
Shouldn't be surprised that Github trips up on the parsing here, but still made me lol